handler/step.go

```diff
@@ -38,6 +38,11 @@ func HandleStartStep(config *config.Config) http.HandlerFunc {
 			return
 		}
 
+		if config.Runner.RunOnHostOnly {
```

Comment:
All this should get set from the platform

Reply:
platform meaning ci manager?